tests/util/test_yaml.py

```diff
@@ -92,6 +92,23 @@ def test_include_dir_list(self):
                 doc = yaml.yaml.safe_load(file)
                 assert sorted(doc["key"]) == sorted(["one", "two"])
 
+    def test_include_dir_list_recursive(self):
+        """Test include dir list yaml."""
+        with tempfile.TemporaryDirectory() as include_dir:
+            temp_dir = tempfile.TemporaryDirectory(dir=include_dir)
+            file_1 = tempfile.NamedTemporaryFile(dir=temp_dir.name,
```

Comment:
Great!

Do you minding including on Temp file in `include_dir` at the same time? Then it's :dolphin: from my side.

```python
file_0 = tempfile.NamedTemporaryFile(dir=include_dir,
...
```